CSharp/README.md

```diff
@@ -206,6 +207,71 @@ var result=2;
 var result = 2;
 ```
 
+## Ordering
+
+### Within a class, struct, or interface, elements must be ordered
+
+1. Constants
+1. Fields
+1. Constructors
+1. Delegates
+1. Events
+1. Properties
+1. Methods
+1. Finalizers (Destructors)

```

Comment:
Should we also include comments about visibility? I remember having a discussion on having most of the public ones first and then the rest.